docs/fr-FR/contributing.md

```diff
@@ -0,0 +1,155 @@
+
+Merci de garder ceci en tête:
+
+* Cela peut prendre plusieurs heures pour que quelqu'un réponde à votre question. Merci d'être patient!
+* Tous ceux impliqués avec QMK fait don de son temps et de son énergie. Nous ne sommes pas payés pour travailler sur ou répondre aux questions concernant QMK.
+* Essayez de poser vos questions de manière à ce qu'elles soient le plus simple à répondre possible. Si vous n'êtes pas sûrs de savoir comment faire, voici quelques bon guides (en anglais):
+  * https://opensource.com/life/16/10/how-ask-technical-questions
+  * http://www.catb.org/esr/faqs/smart-questions.html
+
+# Aperçu du projet
+
+QMK est majoritairement écrit en C, avec quelques fonctions et parties spécifiques écrites en C++. Il est destiné aux processeurs intégrés que l'on trouve dans des clavier, particulièrement AVR ([LUFA](http://www.fourwalledcubicle.com/LUFA.php)) et ARM ([ChibiOS](http://www.chibios.com)). Si vous maîtrisez déjà la programmation sur Arduino, vous trouverez beaucoup de concepts et de limitations familiers. Une expérience préalable avec les Arduino n'est pas nécessaire à contribuer avec succès à QMK.
+
+<!-- FIXME: We should include a list of resources for learning C here. -->
+
+# Où trouver de l'aide?
+
+Si vous avez besoin d'aide, vous pouvez [ouvrir une issue](https://github.com/qmk/qmk_firmware/issues) ou [un chat sur Discord](https://discord.gg/Uq7gcHh).
+
+# Comment contribuer?
+
+Vous n'avez encore jamais contribué à un projet open source? Vous vous demandez comment les contributions dans QMK fonctionnent? Voici un aperçu rapide!
+
+0. Enregistrez-vous sur [GitHub](https://github.com).
+1. Définissez une keymap à contribuer, [trouvez une issue](https://github.com/qmk/qmk_firmware/issues) que vous souhaitez corriger, ou [une fonction](https://github.com/qmk/qmk_firmware/issues?q=is%3Aopen+is%3Aissue+label%3Afeature) que vous voulez ajouter.
+2. Créez un fork sur le dépôt associé avec une issue sur votre compte GitHub. Cela veut dire que vous allez avoir une copie du dépôt sous `your-GitHub-username/qmk_firmware`.
+3. Clonez le dépôt sur votre macine locale en utilisant `git clone https://github.com/github-username/repository-name.git`.
+4. Si vous travaillez sur une nouvelle fonctionnalité, pensez à ouvrir une issue pour parler avec nous du travail que vous souhaitez démarrer.
+5. Créez une nouvelle branche pour votre correctif en utilisant `git checkout -b branch-name-here`.
+6. Faites les changements nécessaires pour corriger le problème ou ajouter la fonctionnalité.
+7. Utilisez `git add insert-paths-of-changed-files-here` pour ajouter les contenus des fichiers modifiés au "snapshot" que git utilise pour gérer l'état du projet, appelé aussi l'index.
+8. Utilisez `git commit -m "Insert a short message of the changes made here"` pour enregistrer le contenu de l'index avec un message descriptif.
```

Comment:
`git commit -m "insérer message"`

Mention that everything has to be in english?

Reply:
I agree, I translated the message, but added at the end that it should be in English